fineract-provider/src/main/resources/db/changelog/tenant/parts/OXY-138_add_on_hold_table_for_savings_account.xml

```diff
@@ -85,7 +85,7 @@
             <column name="code" value="WITHDRAWAL_WITH_LIMIT_SAVINGSACCOUNT_CHECKER"/>
             <column name="entity_name" value="SAVINGSACCOUNT"/>
             <column name="action_name" value="WITHDRAWAL_WITH_LIMIT_CHECKER"/>
-            <column name="can_maker_checker" valueBoolean="false"/>
```

Comment:
I think you need to change the changeset  id on all these edits

Reply:
I just revert it so old one and new one will be same for oxygen

Reply:
fixed